roles/internal/activemq/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Clean activemq path
```

Comment:
What is this task meant to do? 

Right now it seems like every time the playbook is run its deleting activemq and starting again. We don't want to do that. We want the playbook to be idempotent, so every time its run it won't make changes if the task has already been run. Perhaps this task can just be removed?